scripts/globals/item_utils.lua

```diff
@@ -133,3 +134,29 @@ xi.item_utils.addTwoItemEffects = function(target, effect1, effect2, power1, pow
         target:addStatusEffect(effect2, power2, 0, duration, 0, power2)
     end
 end
+
+xi.item_utils.removeMultipleEffects = function(target, effects, count, random)
+    if random == 1 then -- randomize which effects get removed
+        local effects = utils.shuffle(effects)
+    end
+
+    if count > 0 then
+        local function removeStatus()
```

Comment:
Would prefer to declare this local function outside of the containing function for visibility.  Should have no impact on your code.